package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vscode-xml",
-  "version": "0.14.0",
+  "version": "0.13.0",
```

Comment:
This undos this commit: https://github.com/redhat-developer/vscode-xml/commit/54553e4c1234d1d9519a88d021c7811f8eb24f85

I think it's because that commit doesnt change the version number in `package.json`

Reply:
Ryan fixes this in https://github.com/redhat-developer/vscode-xml/pull/293